packages/components/src/components/radio-button-wrapper/radio-button-wrapper-utils.ts

```diff
@@ -1,9 +1,11 @@
 export const changeHandler = ({ target }: Event): void =>
-  // workaround for Safari >= 15.5 which stopped re-rendering slotted input type radio upon removing checked attribute
-  document.querySelectorAll(`input[type=radio][name=${(target as HTMLInputElement).name}]`).forEach((radio) => {
-    radio.setAttribute('hidden', '');
-    radio.removeAttribute('hidden');
-  });
+  // workaround for Safari >= 15.5 -> attribute 'checked' is only set as property and not as attribute, this leads to rendering issues with keyboard navigation
```

Comment:
And also chrome ;)

Reply:
Actually Chrome is fine, only the previously used workaround no longer worked for Chrome. I can add it though if needed?